src/lumen_anndata/controls.py

```diff
@@ -0,0 +1,154 @@
+import asyncio
+
+from io import BytesIO
+
+import cellxgene_census
+import panel as pn
+import param
+import s3fs
+
+from lumen.ai.controls import SourceControls
+
+
+class CellXGeneSourceControls(SourceControls):
```

Comment:
How do you use it? Do we need to parameterize it at the lumen level so you can easily swap it in?